test/Microsoft.DotNet.ApiDiff.Tests/Diff.Operators.Tests.cs

```diff
@@ -0,0 +1,649 @@
+                      }
+                  }
+                """);
+    }
+
+    [Fact]
+    public void TestLeftShiftOperator()
+    {
+        RunTest(beforeCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                    }
+                }
+                """,
+                afterCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                        public static MyClass operator <<(MyClass a, int shift) { throw null; }
+                    }
+                }
+                """,
+                expectedCode: """
+                  namespace MyNamespace
+                  {
+                      public class MyClass
+                      {
+                +         public static MyClass operator <<(MyClass a, int shift) { throw null; }
+                      }
+                  }
+                """);
+    }
+
+    [Fact]
+    public void TestRightShiftOperator()
+    {
+        RunTest(beforeCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                    }
+                }
+                """,
+                afterCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                        public static MyClass operator >>(MyClass a, int shift) { throw null; }
+                    }
+                }
+                """,
+                expectedCode: """
+                  namespace MyNamespace
+                  {
+                      public class MyClass
+                      {
+                +         public static MyClass operator >>(MyClass a, int shift) { throw null; }
+                      }
+                  }
+                """);
+    }
+
+
+    [Fact]
+    public void TestImplicitOperator()
+    {
+        RunTest(beforeCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                    }
+                }
+                """,
+                afterCode: """
+                namespace MyNamespace
+                {
+                    public class MyClass
+                    {
+                        public static implicit operator MyClass(int value) { throw null; }
+                    }
+                }
+                """,
+                expectedCode: """
+                  namespace MyNamespace
+                  {
+                      public class MyClass
+                      {
+                +         public static implicit operator MyClass(int value) { throw null; }
+                      }
+                  }
+                """);
+    }
+
+    [Fact]
+    public void TestExplicitOperator()
```

Comment:
@ericstj this test passes in my local machine (should pass in the CI too, I just pushed it).

It's the next test (explicit checked) that fails.

Reply:
That other one should be fixed now with `4.14.0-2.25111.1` of roslyn.  You could try updating AsmDiff to this version - it's safe to do so just for our tool projects / tests.  Don't update tasks, libs, or rest of SDK to that version though.

Reply:
Fortunately, https://github.com/dotnet/sdk/pull/46546 was just merged so I don't need to update the packages manually.

Reply:
Passing now!  Same with the next test, no longer skipped.